concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -253,6 +253,7 @@ resources:
     type: git
     source:
       uri: https://github.com/alphagov/paas-compose-broker
+      branch: creation_of_mongo_db_v2
```

Comment:
:white_small_square: reminder that this still needs doing, but it would be fine for the TMP commit to undo it to point at HEAD of the branch for testing.